packages/data/addon/serializers/metadata/bard.ts

```diff
@@ -59,31 +65,30 @@ export type RawColumnFunctionArguments = {
   [k: string]: RawColumnFunctionArgument;
 };
 
-export type RawColumnFunctionArgument = {
-  type: 'enum' | 'dimension';
-  defaultValue?: string;
-  values?: ColumnFunctionParametersValues;
-  dimensionName?: string;
-  description?: string;
-};
+export type RawColumnFunctionArgument =
+  | { type: 'enum'; defaultValue?: string | null; values: ColumnFunctionParametersValues; description?: string }
+  | { type: 'dimension'; defaultValue?: string | null; dimensionName: string; description?: string };
 
 type RawTimeGrainPayload = {
   name: string;
   longName: string;
   description?: string;
+  retention?: string;
   dimensions: RawDimensionPayload[];
   metrics: RawMetricPayload[];
 };
 
-type RawTablePayload = {
+export type RawTablePayload = {
   timeGrains: RawTimeGrainPayload[];
   name: string;
   longName: string;
-  description: string;
-  category: string;
+  description?: string;
+  category?: string;
 };
 
 export default class BardMetadataSerializer extends EmberObject implements NaviMetadataSerializer {
+  private namespace = '_fili_generated_';
```

Comment:
This name might give a little more context:
```suggestion
  private namespace = 'normalizer-generated_';
```